orchagent/pfcwdorch.cpp

```diff
@@ -491,9 +491,11 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::handleWdNotification(swss::Notifi
         {
             throw runtime_error("Unknown PFC WD action");
         }
+        entry->second.handler->initCounters();
```

Comment:
>initCounters [](start = 31, length = 12)

What is the purpose of this PR? refactor or fix?
Is the behavior same as before?

Reply:
It's actually a fix. First assumption was that counters will be the same for all handlers. But for example ACL handler needs to count dropped packets in corresponding rule.